Project_source/app/src/main/java/android/clicker/school_live_simulator/GuitarState.kt

```diff
@@ -0,0 +1,26 @@
+package android.clicker.school_live_simulator
+
+/**
+ * @author  Nikita Dimitryuk niksvob@mail.ru
+ */
+abstract class GuitarState {
+    protected abstract val price: Int
+    protected val available_courses: ArrayList<String> = arrayListOf()
```

Comment:
Pay attention! Not String but Song

Reply:
Actually, in UML file there is exactly String type in GuitarState class. In GuitarCourseState class there is "available_playlist: List<Song>"

Reply:
@Ferrociraptus why do we have String type in this list for GuitarState and Delivery type for BicycleState is that true?

Reply:
@Bazalii @nikitochka-akaka 
Yes, it should be String type, and it should contain Names of Course classes.
But the better way it is using type KClass<*>, not String. I should mark that both ways will be work (but if we will be used KClass in the future will be written less amount of code)

Sorry, guys, that I didn't note this moment in UML.

Reply:
> @Bazalii @nikitochka-akaka
> Yes, it should be String type, and it should contain Names of Course classes.
> But the better way it is using type KClass<*>, not String. I should mark that both ways will be work (but if we will be used KClass in the future will be written less amount of code)
> 
> Sorry, guys, that I didn't note this moment in UML.

@Ferrociraptus, why don't we add Enum Class with Courses like for deliveries and webtasks? In the future, we'll have to use these values for decreasing Player.money and it will give us an opportunity to track available courses in same way as for deliveries for BicycleState(or webtasks for ComputerState)

Reply:
Fixed for KClass